### DISCUSSION TITLE
Support for service users

### DISCUSSION THREAD

**Discussion by reinhard-brandstaedter** [Dynatrace/dynatrace-configuration-as-code#1585] [Ideas]
Currently there isn't even a UI to create service users in a Dynatrace Account.
While we can create them via API only, we cannot create them via monaco configuration.

IMO this would be a logical step to manage service users together with other configuration such as workflows or automations.

Are there any plans to support the creation/management of service-users with monaco?

**Comment by warber** [comment]
hi @reinhard-brandstaedter,

Yes, support for service users is on the roadmap.
Implementation has not yet started, but is definitely on the list of topics the team will focus on in the near future.
